daemons/controld/controld_callbacks.c

```diff
@@ -387,10 +387,10 @@ peer_update_callback(enum pcmk__node_update type, pcmk__node_status_t *node,
 gboolean
 crm_fsa_trigger(gpointer user_data)
 {
-    crm_trace("Invoked (queue len: %d)",
-              g_list_length(controld_globals.fsa_message_queue));
+    crm_trace("Invoking FSA (queue len: %u)",
+              controld_fsa_message_queue_length());
     s_crmd_fsa(C_FSA_INTERNAL);
-    crm_trace("Exited  (queue len: %d)",
-              g_list_length(controld_globals.fsa_message_queue));
+    crm_trace("Exited FSA (queue len: %u)",
+              controld_fsa_message_queue_length());
     return TRUE;
```

Comment:
While you're in the area, this should return `G_SOURCE_CONTINUE` - our documentation for this is confusing but if you trace it through, you'll see that's what `g_source_set_callback` expects.  Speaking of things we are inconsistent about...